sdk/avs/Azure.ResourceManager.Avs/test-resources.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
it seems this is not in use?

Reply:
yes, would like to have it as a placeholder ,if in future we are deploying test resources using arm template

Reply:
May I know what test resources have to be created manually beforehand?
Basically, the test resources are supposed to be created using .NET SDK in the test or during the test setup, so that it's easier to re-record the tests by anyone.
For instance, test resources are created with random name during test setup in [this](https://github.com/Azure/azure-sdk-for-net/blob/9ac52a165101b727ff606dd50dd82f771ec7c710/sdk/sqlvirtualmachine/Azure.ResourceManager.SqlVirtualMachine/tests/Scenario/SqlVmGroupTests.cs#L28)

Reply:
OK then